spaces/domains.json

```
@@ -234,4 +234,5 @@
   "vote.honeydao.com": "honeydao.eth",
   "gov.kumatoken.com": "kumainu.eth",
   "gov.beluga.fi": "belugaprotocol.eth"
+  "vote.bitheist.crypto":"bitheist.eth"
```

A comma is missing at the end of the line above your sub-domain. Please also add a space between sub-domain and ENS. Thank you.
```suggestion
  "gov.beluga.fi": "belugaprotocol.eth",
  "vote.bitheist.crypto": "bitheist.eth"
```